.github/workflows/stale-pr.yml

```diff
@@ -0,0 +1,21 @@
+name: "Close stale pull requests"
+on:
+  schedule:
+    - cron: "23 * * * *" # https://crontab.guru/#23_*_*_*_*
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-pr-message: 'This PR was marked stale due to lack of activity. It will be closed in 14 days.'
+          close-pr-message: 'Closed as inactive. Feel free to reopen if this PR is still being worked on.'
+          days-before-pr-stale: 14
+          days-before-pr-close: 14
+          any-of-issue-labels: needs-author-feedback
```

Comment:
will we have to manually remove the `needs-author-feedback` label in order to prevent the issue from going stale and getting closed due to our non-responsiveness?

Reply:
yes, or you can just add some comments say `the issue is not stale`.